### PR TITLE
Added custom logos by volume mounts documentation

### LINKED ISSUE #1066: Docs on how to add custom AWX logo
### Please confirm the following

- [X] I agree to follow this project's [code of conduct](https://docs.ansible.com/ansible/latest/community/code_of_conduct.html).
- [X] I have checked the [current issues](https://github.com/ansible/awx-operator/issues) for duplicates.
- [X] I understand that AWX Operator is open source software provided for free and that I might not receive a timely response.

### Feature Summary

Users may wish to swap out the AWX logo for their own. It would be nice to have docs (and maybe an operator change) to allow for this.

See this PR on how to change awx logo for the older docker based deployments. This approach would be need to be updated to work with k8s deployments.

https://github.com/ansible/awx/pull/12265

### PR DESCRIPTION
**Pull request by YaronL16**
##### SUMMARY
Added documentation regarding customizing the login and header logos via custom volume mounts.

Fixes #1066

##### ISSUE TYPE
 - Bug, Docs Fix or other nominal change

##### ADDITIONAL INFORMATION
Similar to the pre-existing section regarding customizing the favicon.

### CODE REVIEW

**Comment by kurokobo** [comment]
Thanks for working on this, LGTM 😃
Please wait for the review by maintainers (I'm not the one but just a community member).

**Review by rooftopcellist — APPROVED** [review]
@YaronL16 thank you for adding this docs!  The steps here look to be correct.

**Pull request merged**
